### ISSUE TITLE
Do local katabatics matter more than we thought?

### ISSUE THREAD

**Issue by adele-morrison** [adele157/easterlies-collaborative-project#76]
Reviewer 2 says: "It may be worth mentioning that the rejection of the katabatic
hypothesis is not so clear for the Ross box, for which the purple bar is closest to the black bar than to the blue bar." (referring to Figure 4).

I've been investigating this a little further and discovered a bug in the Figure 4 code, where we were doing the time average first before the area integration of the SWMT. I have fixed this now and it does change the results somewhat. Before we were also using a single density threshold to compute the SWMT across to use in the bar plot. This resulted in a very noisy time series and not particularly robust numbers even after the time averaging on the bar plot.

I have now changed the method to average the SWMT over densities corresponding to control SWMT magnitudes that are between 30% and 70% of the control peak SWMT. This is a much more robust quantity.

This is what Figure 4 now looks like:
<img width="850" alt="Screen Shot 2023-04-04 at 10 50 56 am" src="https://user-images.githubusercontent.com/8506963/229658362-7e01df44-d3b9-4b2c-ac8a-038223544882.png">

Across all regions, the WIND+_no_local case has less DSW formation than the WIND+ case. I think before when we looked at these results, we were expecting that the purple bars would be identical to the control and were so surprised that they weren't that we overlooked that the no_local case is not identical to the WIND+ case.

For reference, here is the full SWMT for each region:

<img width="850" alt="Screen Shot 2023-04-04 at 10 51 19 am" src="https://user-images.githubusercontent.com/8506963/229658934-25cb1877-f52b-403c-8831-fda3b10519cc.png">

There is the complication that when we don't apply the wind anomaly in the no_local experiment, we are losing both the katabatics and the sea ice advection in those local regions.

In summary, I think we should modify the results a little to say that local katabatics do play a role in increasing the DSW formation (we could put an upper bound on that which looks like <50% from the bar plot above), but that they are not enough to explain the full change and that the sea ice mechanism is responsible for the rest of the DSW increase.

Thoughts?

**Comment by AndyHoggANU** [comment]
Hi @adele157  - Had a look through this, and I agree. Nice catch.
I guess we should write the response part to this first, check we agree, and then alter the manuscript?

**Comment by adele157** [comment]
Yep let's do it.
